Source/WebCore/platform/graphics/gstreamer/GStreamerCommon.cpp

```diff
@@ -951,6 +951,43 @@ GstElement* makeGStreamerBin(const char* description, bool ghostUnlinkedPads)
     return bin;
 }
 
+#if USE(GSTREAMER_WEBRTC)
+static ASCIILiteral webrtcStatsTypeName(int value)
+{
+    switch (value) {
+    case GST_WEBRTC_STATS_CODEC:
+        return "codec"_s;
+    case GST_WEBRTC_STATS_INBOUND_RTP:
+        return "inbound-rtp"_s;
+    case GST_WEBRTC_STATS_OUTBOUND_RTP:
+        return "outbound-rtp"_s;
+    case GST_WEBRTC_STATS_REMOTE_INBOUND_RTP:
+        return "remote-inbound-rtp"_s;
+    case GST_WEBRTC_STATS_REMOTE_OUTBOUND_RTP:
+        return "remote-outbound-rtp"_s;
+    case GST_WEBRTC_STATS_CSRC:
+        return "csrc"_s;
+    case GST_WEBRTC_STATS_PEER_CONNECTION:
+        return "peer-connection"_s;
+    case GST_WEBRTC_STATS_TRANSPORT:
+        return "transport"_s;
+    case GST_WEBRTC_STATS_STREAM:
+        return "stream"_s;
+    case GST_WEBRTC_STATS_DATA_CHANNEL:
+        return "data-channel"_s;
+    case GST_WEBRTC_STATS_LOCAL_CANDIDATE:
+        return "local-candidate"_s;
+    case GST_WEBRTC_STATS_REMOTE_CANDIDATE:
+        return "remote-candidate"_s;
+    case GST_WEBRTC_STATS_CANDIDATE_PAIR:
+        return "candidate-pair"_s;
+    case GST_WEBRTC_STATS_CERTIFICATE:
+        return "certificate"_s;
+    }
+    return nullptr;
```

Comment:
Do we want to `ASSERT_NOT_REACHED` here and most likely to `return { }`.

Reply:
Yes for the ASSERT_NOT_REACHED i guess, but the return is good as-is imho, ASCIILiteral can be built from a nullptr afaics